extension.js

```diff
@@ -48,7 +48,7 @@ const VSCodeSearchProvider = new Lang.Class({
     this.id = 'VSCodeProjects';
     this.appInfo = {
       get_name: function () { return 'vscode-search-provider'; },
-      get_icon: function () { return Gio.icon_new_for_string("/usr/share/icons/visual-studio-code.png"); },
+      get_icon: function () { return Gio.icon_new_for_string("code"); },
```

Comment:
"code" doesn't work for Arch Linux.
"visual-studio-code" is the identifier here.